### PR TITLE
Add tests for 'rad diff propose'

### PR DESCRIPTION
**Pull request by geigerzaehler**
Add tests for 'rad diff propose'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,16 +52,19 @@ into version control.
 ### End-to-end Tests
 
 The end-to-end test suite is run with `stack test :e2e`. It requires you to
-start up an IPFS test network with
+start up an IPFS test network and the Raicle daemon.
```
Comment (jkarni) [inline README.md]:
```suggestion
start up an IPFS test network and the radicle daemon.
```

File: README.md [hunk 2]
```diff
@@ -52,16 +52,19 @@ into version control.
...
 
     docker-compose -f test/docker-compose.yaml up -d ipfs-test-network
+    RAD_IPFS_API_URL=http://localhost:19301 stack exec -- radicle-daemon --filePrefix test
```
Comment (jkarni) [inline README.md]: Isn't `--file-prefix` the more common option style?

Comment (geigerzaehler) [reply README.md]: Yes, but the current implementation requires `--filePrefix`. I’ll change that in an upcoming commit.

File: test/e2e/DiffAppTest.hs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    ) where
+
+import           Protolude
+
+import           Test.E2ESupport
+
+test_diff_propose :: TestTree
+test_diff_propose = testCaseSteps "diff propose" $ \step -> do
+    step "Init project"
+    _ <- runTestCommand "rad-create-keys" []
```
Comment (jkarni) [inline test/e2e/DiffAppTest.hs]: This is all really cool!

**Pull request merged**